lib/remap-functions/src/encode_json.rs

```diff
@@ -0,0 +1,154 @@
+use remap::prelude::*;
+
+#[derive(Clone, Copy, Debug)]
+pub struct EncodeJson;
+
+impl Function for EncodeJson {
+    fn identifier(&self) -> &'static str {
+        "encode_json"
+    }
+
+    fn parameters(&self) -> &'static [Parameter] {
+        &[Parameter {
+            keyword: "value",
+            accepts: |_| true,
+            required: true,
+        }]
+    }
+
+    fn compile(&self, mut arguments: ArgumentList) -> Result<Box<dyn Expression>> {
+        let value = arguments.required("value")?.boxed();
+
+        Ok(Box::new(EncodeJsonFn { value }))
+    }
+}
+
+#[derive(Debug, Clone)]
+struct EncodeJsonFn {
+    value: Box<dyn Expression>,
+}
+
+impl Expression for EncodeJsonFn {
+    fn execute(&self, state: &mut state::Program, object: &mut dyn Object) -> Result<Value> {
+        let value = self.value.execute(state, object)?;
+        match serde_json::to_string(&value) {
+            Ok(value) => Ok(value.into()),
+            Err(error) => panic!("unable encode to json: {}", error),
```

Comment:
@FungusHumungus is this ok for remap function?

Reply:
I can't see any reason why this would ever error, so theoretically it should be ok. However, I don't think it would hurt to return `Err` instead, just in case.

Reply:
Oh, I see you discussed this already. I'm fine with the panic! as it really should never happen and does allow us to make this infallible as Jesse suggested. 

A comment here to explain the reasoning why this will never occur would be useful.

Reply:
I added comment and changed `panic!` to `unreachable!` what should make the code more clear.